inc/toolbox.class.php

```diff
@@ -2723,11 +2743,11 @@ static function convertTagToImage($content_text, CommonDBTM $item, $doc_data = [
     *
     * @since 9.2
     *
-    * @param $content_html   html content of input
-    * @param $force_update   force update of content in item (false by default)
+    * @param string    $content_html    html content of input
+    * @param boolean   $force_update    force update of content in item (false by default)
     *
-    * @return html content
-   **/
+    * @return mixed|string
```

Comment:
It will always return string if correctly used passing a strig for first parameter.

Reply:
"if correctly used" so this could return mixed values